.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: test
+on: [push, pull_request]
```

Comment:
Also, if we want to go ahead with this, let's limit this to just `pull_request` to avoid running this twice.

Reply:
push would be mostly just on merging, right? but  agree that `pull_request` is just enough.

Reply:
merging and also just pushing to own repos before sending a PR.